### ISSUE TITLE
1.0.4 broke my gulp/sprity setup

### ISSUE THREAD

**Issue by jbruggem** [sprity/sprity#13]
Hi,

I use sprity with gulp in a pretty standard way (pretty much what the doc says) and upgrading to 1.0.4 breaks my gulp process: instead of creating a sprite, the task now just copies all the input files in the output folder. 

Here is an anonymized version of the gulp task definition:

```js

gulp.task('sprites', function () {
    var pathImg = 'path/to/src';
    var pathIcons = pathImg+'/icons';
    var rootBuild = 'output/root/folder';
    var pathCompiledCss = 'path/compiled/css';
    var pathCompiledImg = 'path/compiled/img';

    return sprity.src({
        src: [
            pathImg+'/img1.png',
            pathIcons+'/img2.png',

            pathIcons+'/folder1/img1-1.png',
            pathIcons+'/folder1/img1-2.png',
            pathIcons+'/folder1/img1-3.png',

            pathIcons+'/folder2/img2-1.png',
            pathIcons+'/folder2/img2-2.png',
            pathIcons+'/folder2/img2-3.png',

            pathIcons+'/folder3/img3-1.png',
            pathIcons+'/folder3/img3-2.png',
            pathIcons+'/folder3/img3-3.png',

        ],
        'processor': 'less',
        // name of the generated image file (mow-sprite.png)
        name: 'sprite-file-name',
        // name of the generated styles file (mow-sprite.less)
        'style': 'sprite-file-name',
        // prefix for the css class names
        // in less, just the name of the class referencing the sprite
        'prefix': 'global-sprite',
        // relative page to the image folder when in the styles file
        'cssPath': path.join('..', pathCompiledImg),

        'orientation': 'binary-tree',
        split:false,

    }).pipe(gulpif('*.png',
        gulp.dest(pathCompiledImg),
        gulp.dest(pathCompiledCss)));
});



```

**Comment by jakeg** [comment]
can confirm this issue with latest.

**Comment by loremipson** [comment]
+1

**Comment by aslansky** [comment]
Should be fixed in 1.0.5

**Comment by tibiciobanu** [comment]
thx, that solved it
